fe/fe-core/src/main/java/com/starrocks/sql/StatementPlanner.java

```diff
@@ -388,8 +388,12 @@ private static void beginTransaction(DmlStmt stmt, ConnectContext session)
         String catalogName = stmt.getTableName().getCatalog();
         String dbName = stmt.getTableName().getDb();
         String tableName = stmt.getTableName().getTbl();
+
         Database db = MetaUtils.getDatabase(catalogName, dbName);
-        Table targetTable = MetaUtils.getTable(catalogName, dbName, tableName);
+        Table targetTable = MetaUtils.getTable(session, db, stmt.getTableName());
```

Comment:
should call MetaUtils.getTableSessionAware()? for loading, should always be aware of tmp table?

Reply:
> should call MetaUtils.getTableSessionAware()? for loading, should always be aware of tmp table?

similar reason as above, we already get db here. yeah, for insert statement, should always be aware of temp table.